src/creature.h

```diff
@@ -186,6 +205,15 @@ class Creature
                                                    double total_dispersion );
 
         /**
+         * Makes an aiming/attack roll for a single projectile attack shot.
+         *
```

Comment:
doxygen_conf.txt has JAVADOC_AUTOBRIEF = YES. If you want "no stop at the end of preamble sentence" then you should set that to NO; the current setting relies on the stop to find the end of the brief description.